actin/local/upload/push_actin_changes_in_actin_repo

```diff
@@ -11,7 +11,7 @@ current_path=$(pwd)
 
 info "Committing and pushing changes in actin crunch resources"
 
-cd "$HOME/hmf/repos/crunch-resources-private"
+cd "$HOME/hmf/repos/actin-resources-private"
 
 git pull
 git commit -m "${commit_message}" actin
```

Comment:
Did you consider removing the top level "actin" dir? I'd think it's fine to keep this, to keep the actual resources separate from dockerfile, but it does look a bit weird. Maybe "resources" would be a better name, since "actin" is implied by the repo name. Could leave for future improvement as well though

Reply:
I did consider that, actually that's how I originally implemented it, but found it awkward to have the Dockerfile/cloudbuild/.gitignore in the same path.  I also thought about calling it resources, maybe that is better. I'm going to defer this decision to once all is working.